core/lib/makeswift/components/card-carousel/card-carousel.makeswift.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  Checkbox,
+  Image,
+  Link,
+  List,
+  Select,
+  Shape,
+  Style,
+  TextInput,
+} from '@makeswift/runtime/controls';
+
+import { CardCarousel, CardCarouselProps } from '@/vibes/soul/primitives/card-carousel';
+import { runtime } from '~/lib/makeswift/runtime';
+
+interface Card {
+  title: string;
+  imageSrc?: string;
+  imageAlt: string;
+  link?: { href?: string; target?: string };
```

Comment:
Seems like we're not using the `target` at all in this component. Is this because the VIBES component doesn't use it?

Reply:
Got it. Let's add an issue to the backlog for supporting this in VIBES